### PR TITLE
FED Assignment 1: Blog Bones 🦴

### PR DESCRIPTION
**Pull request by ginabak**
# TL;DR 🚀

Gina's front-end structure (aka **blog 🦴s**)for her Rails Blog App! 🚀

# What does this PR do? 

### Checklist ✅ from [assignment specs](https://shopify.instructure.com/courses/329/assignments/1957?module_item_id=5640)

- [x] Create a new react app using create-react-app
- [x] Break your app into Components
   Created *7* Components: 
         1. BlogPost
         2. HomePage
         3. LatestBlogPost
         4. NavBar
         5. SmallBlogPost
         6. StaticFooter
          7. StaticHeader

- [x] Challenge: componentize further (isolation and reusability)
- [x] Each component must be in its folder inside a components folder in src
- [x] Each component must have an index.js file set to  export default the component
- [x] A rough layout of your blog that can be seen by launching your web app
- [x] Your code is available on the Dev Degree Github Organization and submitted to this Assignment as a PR.

# 🎉*EXTRA* 🎉 things to note

- Implemented use of `Router` for `Links` for the `NavBar` Component
- Notice that `NavBar` Component is functional (no class) ✌️
- Original Logo
- React Animation 🧶(see the bounce | part of the `StaticFooter` Component) installed `react-animations`

### CODE REVIEW

File: gina-blog-app/src/components/BlogPost/BlogPost.js [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+        <div className='post-layout'>
+          <Link to='/' className='category-name'>category</Link> | 19 Mar 2020
+          <h1 className='post-name'>POST-TITLE</h1>
+          <h4>
+            by:  
+            <Link to='/' className='author-name'> blog author</Link>
+          </h4>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit ...</p>
+        </div>
       </div>
```
Comment (derekshopify) [inline gina-blog-app/src/components/BlogPost/BlogPost.js]: Let's be more strict about when we use div's if we have a more semantic alternative. Article makes the most sense here.

File: gina-blog-app/src/components/HomePage/HomePage.js [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
-        <SmallBlogPost />
+        <Router>
+          <LatestBlogPost/>
+          <div className='grid-container'>
+            <div className='grid-item'><SmallBlogPost /></div>
+            <div className='grid-item'><SmallBlogPost /></div>
+            <div className='grid-item'><SmallBlogPost /></div>
+            <div className='grid-item'><SmallBlogPost /></div>
+            <div className='grid-item'><SmallBlogPost /></div>
+            <div className='grid-item'><SmallBlogPost /></div>
```
Comment (derekshopify) [inline gina-blog-app/src/components/HomePage/HomePage.js]: Could these be sections instead of divs?

File: gina-blog-app/src/components/LatestBlogPost/LatestBlogPost.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ class LatestBlogPost extends Component {
   render() {
     return(
       <div className='latest-blog-post-position'>
-        <img className='latest-post-image' src={ require('./images/boxedwater.jpeg')}></img>
+        <img className='latest-post-image' src={ require('./images/plant.png')}></img>
         <BlogPost />
       </div>
```
Comment (derekshopify) [inline gina-blog-app/src/components/LatestBlogPost/LatestBlogPost.js]: Article instead of div
